### PR TITLE
Highlights: Fixes and Improvements

### PR DESCRIPTION
**Pull request by nidzix**
SD-1244
+ Add ui for managing highlights configurations
+ Fix styling issues with highlight dropdowns
+ Improve hilight inclusion list directive for content items

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

File: client/app/scripts/superdesk/ui/ui.js [hunk 1]
```diff
@@ -252,6 +252,29 @@ define([
...
 	}
 
+	DropdownPositionDirective.$inject = ['$document'];
+	function DropdownPositionDirective($document) {
+		return {
+			link: function(scope, element) {
+
+				var tolerance = 250;
+
+				element.bind('click', function(event) {
```
Comment (petrjasek) [inline client/app/scripts/superdesk/ui/ui.js]: not sure here, but don't you have to unbind it?

**Comment by petrjasek** [comment]
:+1:

File: client/app/scripts/superdesk/ui/ui.js (hunk 1 above)
Comment (nidzix) [inline client/app/scripts/superdesk/ui/ui.js]: I think that only those handlers attached to the element **diferent than the one linked to directive**, should be unbinded. For example if I attach `click` event to `document`, here,  in this directive, I would need to unbind it.

Comment (petrjasek) [inline client/app/scripts/superdesk/ui/ui.js]: ok couldn't find it in docs so I'm just checking

**Pull request merged**